server/app/views/style/Styles.java

```diff
@@ -699,6 +699,36 @@ public final class Styles {
   public static final String SELF_END = "self-end";
   public static final String SELF_CENTER = "self-center";
   public static final String SELF_STRETCH = "self-stretch";
+  public static final String SPACE_X_0 = "space-x-0";
+  public static final String SPACE_X_0_5 = "space-x-0.5";
+  public static final String SPACE_X_1 = "space-x-1";
+  public static final String SPACE_X_1_5 = "space-x-1.5";
+  public static final String SPACE_X_2 = "space-x-2";
+  public static final String SPACE_X_2_5 = "space-x-2.5";
+  public static final String SPACE_X_3 = "space-x-3";
+  public static final String SPACE_X_3_5 = "space-x-3";
+  public static final String SPACE_X_4 = "space-x-4";
+  public static final String SPACE_X_5 = "space-x-5";
+  public static final String SPACE_X_6 = "space-x-6";
+  public static final String SPACE_X_7 = "space-x-7";
+  public static final String SPACE_X_8 = "space-x-8";
+  public static final String SPACE_X_9 = "space-x-9";
+  public static final String SPACE_X_10 = "space-x-10";
+  public static final String SPACE_Y_0 = "space-y-0";
+  public static final String SPACE_Y_0_5 = "space-y-0.5";
+  public static final String SPACE_Y_1 = "space-y-1";
+  public static final String SPACE_Y_1_5 = "space-y-1.5";
+  public static final String SPACE_Y_2 = "space-y-2";
+  public static final String SPACE_Y_2_5 = "space-y-2.5";
+  public static final String SPACE_Y_3 = "space-y-3";
+  public static final String SPACE_Y_3_5 = "space-y-3";
+  public static final String SPACE_Y_4 = "space-y-4";
+  public static final String SPACE_Y_5 = "space-y-5";
+  public static final String SPACE_Y_6 = "space-y-6";
+  public static final String SPACE_Y_7 = "space-y-7";
+  public static final String SPACE_Y_8 = "space-y-8";
+  public static final String SPACE_Y_9 = "space-y-9";
+  public static final String SPACE_Y_10 = "space-y-10";
```

Comment:
note (non-blocking): i'm surprised these work, when we generated this file we intended to include all available classes

Reply:
aha